src/main/resources/com/atlassian/jira/cloud/jenkins/configuration/config.jelly

```diff
@@ -111,49 +111,49 @@
                             </f:section>
                         </div>
 
-                        <f:section title="Builds and deployments">
-                            <div class="help-area">
-                                <div class="help display-block">
-                                    <p>Configure what data is sent to connected Jira sites.</p>
-                                    <ul>
-                                        <li>
-                                            If you select the options below, build and deployment data will be sent to all connected Jira sites whenever this server runs pipelines that match criteria you specify.
-                                        </li>
-                                        <span class="or-separator">or</span>
-                                        <li>
-                                            If you don’t select the options below, you can setup individual pipelines to send data to Jira using instructions in their Jenkinsfiles. See <a href="https://plugins.jenkins.io/atlassian-jira-software-cloud" target="_blank">https://plugins.jenkins.io/atlassian-jira-software-cloud</a> for more information.
-                                        </li>
-                                    </ul>
-                                </div>
-                            </div>
-
-                            <!-- Auto Builds -->
-                            <f:entry>
-                                <f:optionalBlock help="/plugin/atlassian-jira-software-cloud/help-autoBuilds.html" name="${config.FIELD_NAME_AUTO_BUILDS}" title="${%Send build data automatically}" checked="${config.getAutoBuildsEnabled()}">
-                                    <f:entry title="${%Build stage regex filter: (Optional)}" field="autoBuildsRegex">
-                                        <f:textbox id="autoBuildsRegex" name="${config.FIELD_NAME_AUTO_BUILDS_REGEX}" value="${config.getAutoBuildsRegex()}" />
-                                        <input type="button" value="Validate Regex" onclick="validateAutoBuildsRegex();" class="validate-regex-btn" />
-                                        <p id="autoBuildsRegexTestResponse"></p>
+                        <div class="advanced-options-accordian" onclick="toggleAdvancedOptions();">
```

Comment:
The div `<div class="advanced-options-accordian"` is new and then wrapped the existing `section` in <`div class="advanced-block advanced-block-default" id="advancedOptionsContainer">`

This is just so we can show/hide the section